admin_guide/securing_builds.adoc

```diff
@@ -0,0 +1,162 @@
+= Securing Builds by Build Strategy
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+
+The build strategies that users are allowed to use when creating `*Builds*` or `*BuildConfigs*`
+can be controlled via an
+link:../architecture/additional_concepts/authorization.html[authorization policy].
+Each build strategy is controlled via access to a build subresource:
+
+.Build strategy subresources
+[cols="1,1",options="header"]
+|===
+
+|Strategy |Resource
+
+|Docker
+|builds/docker
+
+|Source-To-Image
+|builds/source
+
+|Custom
+|builds/custom
+
+|===
+
+By default, project administrators (`admin` role) and project editors (`edit` role) are granted
+permission to use all strategies (Docker, Source-to-Image, and Custom).
+
+
+=== Disabling a build strategy globally
+To prevent access to a particular build strategy globally, login as cluster administrator and edit
+each of the default roles:
+
+----
+$ oadm edit clusterrole admin
+$ oadm edit clusterrole edit
+----
+
+For each role, remove the line that corresponds to the resource of the strategy to disable:
+
+.Disable the Docker build strategy
+=====
+
+----
+kind: ClusterRole
+metadata:
+  name: admin
+...
+
+rules:
+- attributeRestrictions: null
+  resources:
+  - builds/custom
+  - builds/docker <1>
+  - builds/source
+  - pods/exec
+  - pods/portforward
+
+...
+
+----
+<1> Delete this line to disable Docker builds globally
+
+=====
+
+Alternatively, the policy can be updated via non-interactive commands.
+
+.Disable the Docker build strategy using non-interactive commands
+====
+
+----
+$ oc get clusterrole admin -o yaml | grep -v "builds\/docker" | \
+  oc replace clusterrole admin -f -
+
+$ oc get clusterrole edit -o yaml | grep -v "builds\/docker" | \
+  oc replace clusterrole edit -f -
+----
+
+====
+
+
+
+=== Allowing only certain users to create builds with a particular strategy
+
+To allow only a set of specific users to create builds with a particular strategy:
+
+1. link:#disabling-a-build-strategy-globally[Remove the build strategy resource from the default admin and edit roles].
+1. Create a separate role for that build strategy
++
+[#create-separate-role]
+.Create a separate role for the docker build strategy
```

Comment:
How hard was this to do?  Having done it, do you have thoughts about what you'd like in a cli command to make this easier?  If so, can you make an issue with your suggestions and tag me?